lib/index.js

```diff
@@ -27,6 +28,7 @@ function Chargehound (apiKey, options) {
     port: options.port || null,
     protocol: options.protocol || CHARGEHOUND_PROTOCOL,
     timeout: options.timeout || CHARGEHOUND_TIMEOUT,
+    env: options.env || CHARGEHOUND_ENV,
```

Comment:
Do we need a null constant? Wouldnt it be default value automatically if options.env is not set?